### PR TITLE
Cm azure web app

### PR DESCRIPTION
**Pull request by xtreampb**
### Step template checklist

- [ ] `Id` should be a **GUID** that is not `00000000-0000-0000-0000-000000000000`
  - **NOTE** If you are modifying an existing step template, please make sure that you **do not** modify the `Id` property *(updating the `Id` will break the Library sync functionality in Octopus)*. 
- [ ] `Version` should be incremented, otherwise the integration with Octopus won't update the step template correctly
- [ ] Parameter names should not start with `$`
- [ ] **Step template parameter names (the ones declared in the JSON, not the script body) should be prefixed with a namespace so that they are less likely to clash with other user-defined variables in Octopus** (see [this issue](https://github.com/OctopusDeploy/Issues/issues/2126)). For example, use an abbreviated name of the step template or the category of the step template).
- [ ] `LastModifiedBy` field must be present, and (_optionally_) updated with the correct author
- [ ] If a new `Category` has been created:
   - [ ] An image with the name `{categoryname}.png` must be present under the `step-templates/logos` folder
   - [ ] The `switch` in the `humanize` function in [`gulpfile.babel.js`](https://github.com/OctopusDeploy/Library/blob/master/gulpfile.babel.js#L92) must have a `case` statement corresponding to it

### CODE REVIEW

File: step-templates/azure-web-app-restart.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
..."]\n$startIfStopped = $OctopusParameters[\"azWebApp.StartIfStopped\"]\n\nWrite-Host \"Checking webapp $webApp status in resource group $resourceGroup\"\n\n$appState = az webapp list --resource-group chrisplayground --query \"[?name=='$webApp'].{state: state, hostName: defaultHostName}\" | ConvertFrom-Json\nif($appState.state -eq \"stopped\")\n{\n    if($startIfStopped) -eq 'true')\n    {\n        Write-Host \"Webapp is not running. Starting...\" -NoNewline\n    \taz webapp start --name $webApp --resource-group $resourceGroup\n        Write-Host \"Done\"\n    }\n    else\n    {\n      Throw \"Webapp is not running.\"\n    }\n}\n\nWrite-Host \"Webapp running, restarting\"\n\nelse\n{\n\tWrite-Host \"Restarting $webApp in resource group $resourceGroup\"\n\taz webapp restart --name $webApp --resource-group $resourceGroup\n}\n\nStart-Sleep -s 5\n\n$appState = az webapp list --resource-group chrisplayground --query \"[?name=='$webApp'].{state: state, hostName: defaultHostName}\" | ConvertFrom-Json\n\nif($appState.state -ne \"running\")\n{\n\tThrow \"Webapp failed to start.  Check the app's activity/error log\"\n}\n\nwrite-host \"Webapp $webApp running. Check at: $($appState.hostName)\"\n"
```
Comment (harrisonmeister) [inline step-templates/azure-web-app-restart.json:13]: Should the resource group be pointing to a hardcoded value here? 

`--resource-group chrisplayground`

File: step-templates/azure-web-app-start.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  "Name": "Azure Web App - Start",
+  "Description": "Starts an azure web app.\n<hr />\n\n*<p>Note This template is designed to run against an azure web app octopus target </p>*\n*<p>Depends on Azure CLI and powershell to be installed on the running machine</p>*",
+  "ActionType": "Octopus.AzurePowerShell",
+  "Version": 1,
+  "CommunityActionTemplateId": null,
+  "Packages": [],
+  "Properties": {
+    "Octopus.Action.Script.ScriptSource": "Inline",
+    "Octopus.Action.Script.Syntax": "PowerShell",
+    "OctopusUseBundledTooling": "True",
```
Comment (MJRichardson) [inline step-templates/azure-web-app-start.json:12]: Could we switch `OctopusUseBundledTooling` to 'False'?  
Same for the step below.

File: step-templates/azure-web-app-restart.json (hunk 1 above)
Comment (xtreampb) [reply step-templates/azure-web-app-restart.json:13]: @harrisonmeister you are absolutly right i didn't scrub that out.  I'll go through these again and make sure the variables are being used.

File: step-templates/azure-web-app-start.json (hunk 2 above)
Comment (xtreampb) [reply step-templates/azure-web-app-start.json]: updated in commit #1164790

File: step-templates/azure-web-app-restart.json (hunk 1 above)
Comment (xtreampb) [reply step-templates/azure-web-app-restart.json]: updated in commit bc42c4b

**Pull request merged**